parallaxinc/Parallax-IDE
# add icons to the application - closes #53

#### What's this PR do?

Adds provided icons to the Chrome application.
#### Where should the reviewer start?

Pull down this branch and reload the application in chrome://extensions.
#### How should this be manually tested?

Pull down this branch and reload the application in chrome://extensions. Verify that the icons display.
#### Any background context you want to provide?

We were using the default icons in the past.
#### What are the relevant tickets?
#53
#### Screenshots (if appropriate)

![Installed application with icons](https://cloud.githubusercontent.com/assets/992373/6995586/8c72715e-db06-11e4-9375-28ea45308c7e.png)
